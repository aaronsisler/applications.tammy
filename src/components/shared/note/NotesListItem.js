import React from 'react';
import PropTypes from 'prop-types';

export default class NotesListItem extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        // cols="75"
        return (
            <div className="notes_list_item">
                <div className="notes_list_item__status_note">
                    <textarea
                        readOnly={true}
                        rows="5"
                        value={this.props.statusNote}
                    >
                    </textarea>
                </div>
            </div>
        );
    }
}

NotesListItem.propTypes = {
    statusNote: PropTypes.string.isRequired,
};
