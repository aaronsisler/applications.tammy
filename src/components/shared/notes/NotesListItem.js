import React from 'react';
import PropTypes from 'prop-types';

export default class NotesListItem extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="notes_list_item">
                <div className="notes_list_item__status_note">
                    <textarea
                        className="textarea"
                        cols="100"
                        readOnly={true}
                        rows="5"
                        value={this.props.statusNote}
                    >
                    </textarea>
                </div>
            </div>
        );
    }
}

NotesListItem.propTypes = {
    statusNote: PropTypes.string.isRequired,
};
