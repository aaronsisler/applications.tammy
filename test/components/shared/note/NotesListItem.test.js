import React from 'react';
import { shallow } from 'enzyme';
import NotesListItem from 'Shared/note/NotesListItem';
import { note } from '../../../fixtures/notes';

describe('NotesListItem', () => {
    let wrapper;
    const { noteMessage } = note;

    beforeEach(() => {
        wrapper = shallow(
            <NotesListItem
                noteMessage={noteMessage}
            />
        );
    })

    it('should render NotesListItem correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
