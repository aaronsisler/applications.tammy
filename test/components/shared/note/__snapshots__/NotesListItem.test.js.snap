// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotesListItem should render NotesListItem correctly 1`] = `
<div
  className="notes_list_item"
>
  <textarea
    readOnly={true}
    rows="5"
    value="First note"
  />
</div>
`;
